### DISCUSSION TITLE
Metrics - did anyone manage to set it up correctly?

### DISCUSSION THREAD

**Discussion by mitsukuri** [XTLS/Xray-core#4189] [Q&A]
I'm trying to follow https://xtls.github.io/en/config/metrics.html and despite doing everything to the letter, I can't for the life of me get any output when accessing http://127.0.0.1:11111/debug/pprof/ and http://127.0.0.1:11111/debug/vars

Did anyone have luck setting it up?

Xray `24.11.30`

**Comment by WhCollar** [comment]
This is also relevant for me, since numerous attempts do not bring results.

**Reply by mitsukuri** [reply]
I only managed to enable it using bare xray core, couldn't do that from 3x-ui panel, which would override the config on its own. I also guess the order of entries in the config matters, but I wasn't smart enough to version control the config with git and track the exact change that finally enabled the metrics.

Also I changed both `metrics_in` and `metrics_out` tags to just `metrics`. Hope that helps!
